--- ethumb-0.1.1.65643/src/lib/client/Makefile.am.orig	2010-07-09 12:46:38.000000000 +0200
+++ ethumb-0.1.1.65643/src/lib/client/Makefile.am	2011-12-13 19:56:04.175326922 +0100
@@ -18,7 +18,7 @@
 libethumb_client_la_DEPENDENCIES = $(top_builddir)/config.h
 libethumb_client_la_LIBADD = \
 	$(top_builddir)/src/lib/libethumb.la \
-	@EDBUS_LIBS@
+	@EDBUS_LIBS@ @EINA_LIBS@ @ECORE_LIBS@
 libethumb_client_la_LDFLAGS = -no-undefined @lt_enable_auto_import@ -version-info @version_info@
 
 # @EVAS_LIBS@ @ECORE_EVAS_LIBS@ @ECORE_FILE_LIBS@ @EDJE_LIBS@ 
